Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Internal/CoreRemoteMongoCollection.swift

```diff
@@ -414,11 +414,13 @@ public class CoreRemoteMongoCollection<T: Codable>: Closable {
      */
     public func watch(
         ids: [BSONValue],
-        delegate: SSEStreamDelegate
+        delegate: SSEStreamDelegate,
+        useCompactEvents: Bool
     ) throws -> RawSSEStream {
         var args = baseOperationArgs
 
         args["ids"] = ids
+        args["useCompactEvents"] = useCompactEvents
 
```

Comment:
I would suggest making this a constant at the top of the file but I know we dont do that throughout the rest of the file